### PR TITLE
New loading screen component

### PR DESCRIPTION
**Pull request by jojopirker**
This is an implementation of a new loading screen component as mentionend in #103.  I also replaced the 
```if (tasks.length == 0)``` with the `isLoading` from `useSWRImmutable`. Also includes a story for storybook, which showcases usage with and without text.

### CODE REVIEW

File: website/src/pages/create/user_reply.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import poster from "src/lib/poster";
 import { Messages } from "src/components/Messages";
 import { TwoColumns } from "src/components/TwoColumns";
 import { Button } from "src/components/Button";
+import { LoadingScreen } from "@/components/Loading/LoadingScreen";
```
Comment (AbdBarho) [inline website/src/pages/create/user_reply.tsx:11]: please use `src` instead of `@`

**Review by AbdBarho — COMMENTED** [review]
Thanks, looks good.

We wanted to refactor the `isLoading` and `task.length` in #104.

I see we merge #110 first and then this one?

File: website/src/components/Loading/Loading.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default {
+  title: "Example/LoadingScreen",
+  component: LoadingScreen,
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+
+const Template = (args) => <LoadingScreen {...args} />; //<><div>text</div><div className="max-w-500 mt-40 z-1000 h-full relative"></div></>;
```
Comment (fozziethebeat) [inline website/src/components/Loading/Loading.stories.jsx:11]: the commented out div can probably be deleted

File: website/src/pages/create/user_reply.tsx (hunk 1 above)
Comment (fozziethebeat) [reply website/src/pages/create/user_reply.tsx:11]: I think it's fine if vs code is doing it.  I'm going to delete this mapping from tsconfig soon so it stops happening.

**Review by fozziethebeat — COMMENTED** [review]
I agree, let's merge in the storybook change and then this.  I'll approve after #110 goes in.

**Review by AbdBarho — APPROVED** [review]

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**